.asf.yaml

```diff
@@ -0,0 +1,10 @@
+github:
```

Comment:
Yaml files allow asf license header using `#` at the beginning of the line.

Reply:
isnt this like '.gitignore' a configuration file...
this file wont be on released jars.

Reply:
The configuration below on the pom disables license checking on files that do not allow license headers. all files that allow should have.

Reply:
done